enterprise/users/src/test/java/io/crate/auth/user/StatementPrivilegeValidatorTest.java

```diff
@@ -479,5 +479,11 @@ public void testPermissionCheckIsDoneOnSchemaAndTableNotOnTableAlias() {
         analyze("select * from doc.users as t");
         assertAskedForTable(Privilege.Type.DQL, "doc.users");
     }
+
+    @Test
+    public void testDecommissionRequiresSuperUserPrivileges() {
+        expectedException.expectMessage("User \"normal\" is not authorized to execute statement");
```

Comment:
Should we include the statement in the message and also tell that it requires super user permissions?

Reply:
cc: @lukas-bernert

Reply:
I guess it would make sense. Then the user immediately know who to turn to.

Reply:
+1 for superuser requirement. why include the statement, isn't it known because this statement resulted in the error?

Reply:
Just a thought. Sometimes if it pops up in stacktraces of applications you're not fully aware what statement was executed. But it's probably overkill so I left it out.